Sources/SemanticIndex/SemanticIndexManager.swift

```diff
@@ -433,13 +421,16 @@ public final actor SemanticIndexManager {
     if Task.isCancelled {
       return
     }
```

Comment:
There's a comment above `Should be kept in sync with schedulePreparationForEditorFunctionality`, is that outdated now?

Reply:
Yes, that’s no longer needed. Thanks!